### DISCUSSION TITLE
corsair icue icon please!

### DISCUSSION THREAD

**Discussion by myodan** [elrumo/macOS_Big_Sur_icons_replacements#1188] [Icon request] [answered]
![dckiclf-d7b6bae1-5188-4dd6-b64c-6401b9c5d7bd](https://user-images.githubusercontent.com/41239679/120635566-9434c180-c4a7-11eb-8d54-f56f1fc62288.png)
please!

**Comment by blaichch** [comment, marked as answer]
I submitted a version. Should be available as soon as it gets verified.

Edit: I add it here too since it hasn't been verified yet.
<img width="512" alt="corsair" src="https://user-images.githubusercontent.com/12624897/120670368-a5260880-c490-11eb-9132-27140c3fe174.png">

**Comment by rodgerwerner** [comment]
Here's my version, without seeing @blaichch's
I will not be submitting this, it's just for here.

![Corsair iCue](https://user-images.githubusercontent.com/23433246/120653579-d1b93080-c44e-11eb-93ad-97297369d5ee.png)
